libcore/src/pedestrian/PedDistributor.cpp

```diff
@@ -154,10 +153,9 @@ bool PedDistributor::Distribute(Building * building) const
             if(allFreePosRoom.count(subroomID) > 0)
                 continue;
             auto possibleSubroomPositions = PossiblePositions(*it_sr.second);
-            shuffle(
-                possibleSubroomPositions.begin(),
-                possibleSubroomPositions.end(),
-                dist->GetGenerator());
+            Random::ShuffleContainer(
+                possibleSubroomPositions.begin(), possibleSubroomPositions.end());
+
```

Comment:
Maybe it would better to call shuffle with a sepcific Generator coming from the random toolset?

Reply:
Shuffle container is the wrapper to that type of shuffling (now with Fisher-Yates Algorithm)